cluster.go

```diff
@@ -1164,6 +1170,32 @@ func (c *ClusterClient) PoolStats() *PoolStats {
 	return &acc
 }
 
+// GetFlowtokenStates returns all flowtoken status.
+func (c *ClusterClient) GetFlowtokenStates() []*flowtoken.Snapshot {
```

Comment:
Looks like this func is unused?

Reply:
User can trace flowtoken states by calling this func periodly.